exasol_integration_test_docker_environment/lib/test_environment/ports.py

```diff
@@ -25,21 +25,31 @@ def check_port_numbers(ports):
     return ports
 
 
-class PortForwarding:
+class PortsType(type):
+    @property
+    def default_ports(self) -> 'PortForwarding':
+        return PortForwarding(database=8888, bucketfs=6583, ssh=22)
+
+    @property
+    def external(self) -> 'PortForwarding':
+        return PortForwarding(database=8563, bucketfs=6583, ssh=22)
+
+    @property
+    def docker(self) -> 'PortForwarding':
+        return PortForwarding(database=8888, bucketfs=6583, ssh=22)
+
+    @property
+    def forward(self) -> 'PortForwarding':
```

Comment:
where is forward used?

Reply:
Currently nowhere.
The initial idea was to use in in `pytest_itde/__init__.py::_bootstrap_db()`. 
But maybe using production code in the pytest plugin is a bad idea? 
Should I remove this?

Reply:
I will remove the unused property.